src/views/menus/AccountMenu/AccountMenu.tsx

```
@@ -56,131 +56,127 @@
-                            {/* <div>
+  return onboardingState === OnboardingState.Disconnected ? (
+    <OnboardingTriggerButton size={ButtonSize.XSmall} />
+  ) : (
+    <$DropdownMenu
+      modal={false}
+      slotTopContent={
+        onboardingState === OnboardingState.AccountConnected && (
+          <div tw="flexColumn gap-1 px-1 pb-0.5 pt-1">
+            <$AddressRow>
+              <AssetIcon
+                logoUrl={'https://verified-assets.fuel.network/images/fuel.svg'}
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)